pages/linux/dnf5.md

```diff
@@ -0,0 +1,37 @@
+# dnf5
+
+> Package management utility for RHEL, Fedora, and CentOS (it replaces dnf, which in turn replaced yum).
+> DNF5 is a complete rewrite of the DNF package manager, written in C++ for improved performance and a smaller size.
+> For equivalent commands in other package managers, see <https://wiki.archlinux.org/title/Pacman/Rosetta>.
+> More information: <https://dnf5.readthedocs.io/>.
+- Upgrade installed packages to the newest available versions:
+
+`sudo dnf5 upgrade`
+
+- Search packages via keywords:
+
+`dnf5 search {{keyword1 keyword2 ...}}`
+
+- Display details about a package:
+
+`dnf5 info {{package}}`
+
+- Install a new package (use `-y` to confirm all prompts automatically):
```

Comment:
```suggestion
- Install a new package (Note: use `-y` to confirm all prompts automatically):
```

Reply:
This is actually identical to the existing `dnf` page. Is this stylistic choice codified?

Reply:
> This is actually identical to the existing `dnf` page. Is this stylistic choice codified?

It isn't documented yet in the style guide (i.e. on which cases we should add a Note), but we have a check for its capitalization and styling in tldr-lint.